x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -268,8 +275,15 @@ export const getJobConfigFromFormState = (
       formState.featureBagFraction && {
         feature_bag_fraction: formState.featureBagFraction,
       },
+      formState.featureProcessors && {
+        feature_processors: formState.featureProcessors,
+      },
       formState.gamma && { gamma: formState.gamma },
       formState.lambda && { lambda: formState.lambda },
+      formState.lossFunction && { loss_function: formState.lossFunction },
```

Comment:
If I clone a job with a non-default `loss_function` such as `huber`, run the job, and then clone it, and now remove the custom `loss_function` from the advanced editor, and switch back to the form editor before running the job, the original `loss_function: huber` is retained. 

![loss_function_clone](https://user-images.githubusercontent.com/7405507/128741809-0323a88b-7090-490f-9421-d533c6adbdad.gif)

Reply:
Good catch! Thanks for testing! This is fixed by 79677d28cecdc918c235f3541819d7e6d3e6153f